step-capstone/src/Components/Map.js

```diff
@@ -0,0 +1,42 @@
+import React, { createRef } from 'react'
+
+/**
+ * Props:
+ * zoom : 13 - higher the number, the more zoomed the map is on center
+ * center: {lat:0.0, lng:0.0} - coordinates for center of map
+ */
+
+
+class Map extends React.Component {
+  
+  componentDidMount() {
+    const loadGoogleMapScript = document.createElement('script');
+    loadGoogleMapScript.src =
+     'https://maps.googleapis.com/maps/api/js?key=AIzaSyAoxC_RWciy_EVsxVImCWk09xFhWf6AwgY&libraries=place';
```

Comment:
You might want to pull out your API key as an environment variable (best) or a constant defined in this file. That way it's clear that's what the url means and you can change the key as needed.